src/celeritas/random/XorwowRngData.cc

```diff
@@ -18,23 +18,21 @@ namespace celeritas
 {
 //---------------------------------------------------------------------------//
 /*!
- * Resize and initialize with the seed stored in params.
+ * Initialize XORWOW states with well-distributed random data.
+ *
+ * cuRAND and hipRAND implement functions that permute the original (but
+ * seemingly arbitrary) seeds given in Marsaglia with 64 bits of seed data. We
```

Comment:
Maybe worth placing the DOI or link to make it a bit easier to access? (Even though this ref is now in the references.bib)

Reply:
Yeah, I was just thinking it would be nice to have a better way to document actual references inline in the code... I'm not sure if there's a good way at all to turn "inline references" into a bibtex document without some hacky preprocessing or something, unless we can insert ```:cite:`foo` ``` in the code and hope the associated reference shows up in the doxygen ...

Reply:
Yeah, I was not too hopeful to have a hyperlink that ends up working on Doxygen to be honest, so I just imagined a DOI (or similar) as plain text. So even if we can't click to be redirected, we can copy/paste it (or select, right click -> Search with Google).

Reply:
Oh I guess a hyperlink would work... I feel like that could be a separate issue where we do this for all our references.